spa_ui/self_service/client/index.html

```diff
@@ -63,8 +63,6 @@
   <script src="/bower_components/patternfly/components/bootstrap-datepicker/dist/js/bootstrap-datepicker.js"></script>
   <script src="/bower_components/patternfly/components/bootstrap-select/dist/js/bootstrap-select.js"></script>
   <script src="/bower_components/patternfly/dist/js/patternfly.js"></script>
-  <script src="/bower_components/patternfly/components/c3/c3.js"></script>
-  <script src="/bower_components/patternfly/components/d3/d3.js"></script>
```

Comment:
Thanks for this part...this is actually causing the nightly appliance to have a diff after we bower install.